python/cuspatial/cuspatial/core/gis.py

```diff
@@ -335,3 +338,40 @@ def polyline_bounding_boxes(poly_offsets, xs, ys, expansion_radius):
     return DataFrame._from_data(
         *cpp_polyline_bounding_boxes(poly_offsets, xs, ys, expansion_radius)
     )
+
+
+def pairwise_linestring_distance(xs1, ys1, offsets1, xs2, ys2, offsets2):
```

Comment:
In the C++ API you have the offsets before the coordinates, which is the opposite of this Python API. It would be nice to keep them consistent.